apps/atomy-q/API/app/Adapters/InsightOperations/GovernanceFactsAdapter.php

```
@@ -0,0 +1,144 @@
+        }
+
+        return 'current';
+    }
+
+    /**
+     * @param Collection<int, VendorEvidence> $evidence
+     * @return array<string, mixed>
+     */
+    private function evidenceFreshness(Collection $evidence): array
+    {
+        $now = CarbonImmutable::now('UTC');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Carbon\CarbonImmutable' in method 'evidenceFreshness'. (undefined)

(StaticAccess)